components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/dao/AppMDAO.java

```diff
@@ -2383,41 +2383,54 @@ public long getAPISubscriptionCountByAPI(APIIdentifier identifier)
      * @return subscription count of apps
      * @throws org.wso2.carbon.appmgt.api.AppManagementException
      */
-    public Map<String, Long> GetSubscriptionCountByApp(String providerName, String fromDate, String toDate, int tenantId)
+    public Map<String, Long> getSubscriptionCountByApp(String providerName, String fromDate, String toDate, int tenantId)
             throws AppManagementException {
 
 
         Connection connection = null;
         PreparedStatement ps = null;
         ResultSet result = null;
+        String sqlQuery;
 
         Map<String, Long> subscriptions = new TreeMap<String, Long>();
 
         try {
             connection = APIMgtDBUtil.getConnection();
 
 
+
             if ("__all_providers__".equals(providerName)) {
-                String sqlQuery =
-                        "SELECT" + "  API.APP_NAME,API.APP_VERSION, API.APP_PROVIDER,COUNT(SUB.SUBSCRIPTION_ID) AS SUB_ID,API.UUID AS uuid"
-                                + " FROM APM_SUBSCRIPTION SUB, APM_APP API, APM_SUBSCRIBER SUBR, APM_APPLICATION  APP"
-                                + " WHERE  API.APP_ID=SUB.APP_ID"
-                                + " AND SUB.APPLICATION_ID=APP.APPLICATION_ID"
-                                + " AND APP.SUBSCRIBER_ID=SUBR.SUBSCRIBER_ID"
-                                + " AND SUBR.TENANT_ID = ?"
-                                + " AND SUB.SUBSCRIPTION_TIME BETWEEN ? AND ?"
-                                + " GROUP BY API.APP_NAME,API.APP_PROVIDER,APP_VERSION ";
+                sqlQuery = "SELECT API.APP_NAME, API.APP_VERSION, API.APP_PROVIDER, "
+                          + "API.UUID AS UUID, COUNT(SUB.SUBSCRIPTION_ID) AS SUB_ID "
+                          + "FROM APM_SUBSCRIPTION SUB, APM_APP API, APM_SUBSCRIBER SUBR, "
+                          + "APM_APPLICATION APP WHERE API.APP_ID = SUB.APP_ID AND "
+                          + "SUB.APPLICATION_ID=APP.APPLICATION_ID AND "
+                          + "APP.SUBSCRIBER_ID=SUBR.SUBSCRIBER_ID AND SUBR.TENANT_ID = ? "
+                          + "AND SUB.SUBSCRIPTION_TIME BETWEEN ";
+                if (!connection.getMetaData().getDriverName().contains("Oracle")) {
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/97624962445a2a9c12448bd5e61bfbba719bf8f3